### DISCUSSION TITLE
FYI, flux-1-schnell works with 3080 10GB VRAM, 32GB RAM

### DISCUSSION THREAD

**Discussion by ETCore7** [Acly/krita-ai-diffusion#1075] [General]
As the title says, it works. Haven't tried the dev version yet. 512x512 generates 4 images, 1014x1024 just one.

Schnell is too slow for live preview here, but in generation mode it is fine. It's a bit slower than SDXL, but the quality is great, and text is rendered correctly most of the time. For example "Protester holds sign with text 'This works great'."

I don't know how a 17GB model works with a 10GB GPU, but it does. 
![image](https://github.com/user-attachments/assets/d307a5db-e2cb-4727-beac-f322b9ec74a8)

Acly and contributers, thank you very much for your fast updates.

**Comment by 1s0n** [comment]
yo how long it took to generate the image, i have the same gpu and ram, just curious

**Comment by ETCore7** [comment]
512x512 in live-mode takes 23 seconds for 1 preview image. 512x512 in generation mode takes 35 seconds, it creates 4 images.

1024x1024 in generation mode takes 30 seconds, it creates 1 image.
![image](https://github.com/user-attachments/assets/a13246f1-90d8-487e-863f-8c8530db97a8)
